### ISSUE TITLE
[ACCESSIBILITY] Touch target sizes should be increased on multiple elements

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1456]
**Describe the accessibility issue**
While using [Android accessibility scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US&gl=US) on the Pixel 4a, I was alerted several elements should have the touch target size increased. Please use the scanner and adjust elements listed. The [WCAG 2.5.5 for Target ](https://www.w3.org/TR/WCAG21/#target-size)size states, the size of target for pointer inputs is at least 44 by 44 CSS pixels. 

**To Reproduce
Steps to reproduce the behavior:**

1. Open  [Android accessibility scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US&gl=US) on the android
2. Open IL app
3. Navigate to notification settings

**Expected behavior**
The scanner will scan and find no accessibility issues

**Actual behavior**
The scanner will note several elements that need touch target size increased. (See video of these from Pixel4a but it is recommended developer downloads this scanner and uses it on an android to see each element that needs to be addressed. 
 
**Screenshots**
https://drive.google.com/file/d/12EUj-0m8_dBF6yg_SETDyDzg-KbmvDt8/view?usp=sharing

**Smartphone (please complete the following information):**
 iPhone XR- iOS 14.6, Pixel 4a- Android 12

Additional context

**Comment by acovher2** [comment]
Closed as this is AAA guideline :(
